node/pkg/reporter/types.go

```diff
@@ -52,7 +44,7 @@ type App struct {
 	Reporters []*Reporter
```

Comment:
not related to this pr, but how about making this into map?

Reply:
hmm any specific benefit?

Reply:
thought you were accessing reporters based on interval somewhere in the code, maybe that's not the case. you can ignore it